components/LoadingPage.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import nwPlusReversed from '../assets/nwplus-reversed.gif';
+
+const LoadingScreenContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  text-align: center;
+`;
+
+const LoadingImage = styled.img`
+  height: 100px;
+  width: 100px;
+  margin: auto;
+`;
+
+const LoadingDiv = styled.div`
```

Comment:
not a blocker but why no `p` or text element? Also prefer label `Container` over `Div` for consistency